Project/DAQ_System/Program/Src/Platform/I2C/Interfaces/ii2c.hpp

```diff
@@ -0,0 +1,32 @@
+/*
+ * ii2c.hpp
+ *
+ *  Created on: Feb 23, 2024
+ *      Author: Nico
+ */
+
+#ifndef II2C_H
+#define II2C_H
+
+#include <cstdint>
+#include <vector>
+
+namespace platform{
+
+class II2C{
+public:
+	static constexpr uint8_t kMaxBytes = 2;
```

Comment:
What is the use-case for `kMaxBytes`?

Reply:
might be wrong but I think it's for I2C transmission since each I2C transmission would require two consecutive I2C messages. one for register address and one for the data to write to the register. If what I said is correct, then I would also consider making it a private variable instead of keeping it public since this is something that the user shouldn't have to be concerned about.

Reply:
It's currently used in the transmitting function. I think it may make more sense to simply derive the amount of bytes to be sent based on the size of the Tx buffer.